packages/template-ui/src/components/ChannelNavBar.vue

```diff
@@ -28,6 +30,9 @@ export default {
     notAtHome() {
       return this.$route.name !== 'Home';
     },
+    showLogo() {
+      return !(this.xs && this.node.ancestors.length);
```

Comment:
Ah here is the answer to the question above. In XS it doesn't show the logo except for the Home.